lua/lspsaga/hover.lua

```diff
@@ -249,7 +250,7 @@ function hover:do_request(args)
       else
         value = result.contents.value
       end
-    elseif vim.tbl_islist(result.contents) then -- MarkedString[]
+    elseif vim.islist(result.contents) then -- MarkedString[]
```

Comment:
need a alias for neovim version islist is in 0.10 for 0.9 still need tbl_islist.